test/terra/primitives/test_sampler.py

```diff
@@ -537,6 +537,18 @@ def test_num_clbits(self, shots):
         bin_probs = quasis.binary_probabilities()
         self.assertDictAlmostEqual(bin_probs, {"0000": 0.5, "0001": 0.5}, delta=1e-2)
 
+    def test_multiple_cregs(self):
+        """Test for the circuit with multipe cregs"""
+        qc = QuantumCircuit(2)
+        cr1 = ClassicalRegister(1, "cr1")
+        cr2 = ClassicalRegister(1, "cr2")
+        qc.add_register(cr1)
+        qc.add_register(cr2)
+        qc.measure(0, 0)
+        qc.measure(1, 1)
+
+        print(Sampler().run(qc, shots=100).result())
```

Comment:
I suggest comparing the result with an expected outcome.

Reply:
Thanks. My fault.